### ISSUE TITLE
[Critical] Role: The element's role is missing or incorrect

### ISSUE THREAD

**Issue by ntallman** [APTrust/dart#625]
## Issue Details

| | |
|---|---|
| **Source Issue ID** | 87779 |
| **Checkpoint** | 4.1 Compatible › Custom Controls (4.1.2.b) |
| **Impact** | Critical |
| **Standards** | wcag2a, wcag2aa, wcag21a, wcag21aa, wcag22a, wcag22aa |
| **Test Area** | 03 New Job Workflow |
| **URL** | <https://aptrust.org/documentation-page/dart-digital-archivists-resource-tool/> |
| **Environment** | Production |
| **Platform** | Windows | Chrome | Firefox |
| **Assistive Technology** | JAWS | NVDA | Magnifier |
| **Method** | Manual |
| **Digital Asset Type** | Desktop Web |
| **Release** | 4.2026 |
| **Date Reported** | 05/12/26 at 5:28 pm |
| **Issue Status** | Open |
| **Unit Type** | Page |
| **Issue Type** | Accessibility |
| **Reported By** | kim.casey@ablr360.com |

## Description

The element's role is missing or is not appropriate for the element's function.
On the Bag Metadata screen, each tag or section is labeled as the "file name (e.g. aptrust-info.txt) expanded same page link".  A more universal indicator of the expand/collapse state would be a down/up arrow icon and the label should be "<name> area expanded/collapsed".  See Create Job User Experience video at 20:35.

## Recommendation

Rule : 
The name, role, value, states, and properties of user interface components MUST be programmatically determinable by assistive technologies.

How To Fix: 
Fix this issue by only using appropriate, valid ARIA roles per the ARIA Recommendation document and the ARIA Authoring Practices document (see References below).

Reference: 
WAI-ARIA Recommendation: https://www.w3.org/TR/wai-aria-1.1/
WAI-ARIA Authoring Practices: https://www.w3.org/WAI/ARIA/apg/

Background: 
Every user interface control must have a role to convey what type of control it is for screen reader and other assistive technology users. Native HTML elements - such as <button>, <a>, <input>, <select> - already have a role, so nothing more needs to be done. If you create a custom version of a native HTML element or a custom control or widget that does not have a native HTML equivalent, you must add the relevant role(s) using ARIA as well as expected keyboard interactions.

## Screenshots

![03_element role](https://raw.githubusercontent.com/APTrust/dart/master/docs/accessibility-audit/screenshots/87779/03_element_role.png)
